geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ShowMetricsCommandIntegrationTest.java

```diff
@@ -164,8 +158,16 @@ public void categoryOptionAbridgesOutput() throws Exception {
     List<String> expectedCategories = Arrays.asList("cluster", "cache", "");
     logger.info("Expecting categories: " + String.join(", ", expectedCategories));
 
-    gfsh.connectAndVerify(server.getEmbeddedLocatorPort(), PortType.locator);
     gfsh.executeAndAssertThat(cmd).tableHasColumnOnlyWithValues("Category",
         expectedCategories.toArray(new String[0]));
   }
+
+  @Test
+  public void getRegionMetricsForPartitionedRegionWithStatistics() throws Exception {
+    String cmd = "create region --name=region2 --type=PARTITION --enable-statistics";
```

Comment:
If `getStatistics` is going to fail the `region.getStatistics()` invocation, should we really want this command to succeed?  Probably beyond the scope of this ticket, but I feel like this `create region` command is invalid and should throw a `UserErrorException`.

Reply:
I think this is actually a valid comment. The "create region --type=PARTITION --enable-statistics" should fail validation also. That's a usability issue we should have a ticket for.

But that would not erase the need for the exception check in the RegionMBeanBridge. We do not know for suer if other type of region would also have this kind of problem.